mollie-payments-for-woocommerce/includes/mollie/wc/gateway/banktransfer.php

```diff
@@ -93,11 +78,11 @@ public function addPaymentArguments (array $args, WC_Order $order)
      * {@inheritdoc}
      *
      * @param WC_Order $order
-     * @param Mollie\Api\Resources\Payment $payment
+     * @param Mollie\Api\Resources\Order $mollie_order
      *
      * @return string
      */
-    protected function getProcessPaymentRedirect(WC_Order $order, Mollie\Api\Resources\Payment $payment)
+    protected function getProcessPaymentRedirect(WC_Order $order, $payment_object)
```

Comment:
This should be `$mollie_order` like in the doc block?

Reply:
Already changed that :)